projects/uitdatabank/models/event-attendanceMode.json

```diff
@@ -0,0 +1,10 @@
+{
+  "title": "event.attendanceMode",
+  "type": "string",
+  "description": "Indicates if the [event](./event.json) is happening online, offline, or both (mixed).",
+  "enum": [
+    "online",
+    "offline",
+    "mixed"
+  ]
+}
```

Comment:
For completeness it would be best to also link this in the `toc.json` under the "Event" group where all event properties are currently listed: https://github.com/cultuurnet/apidocs/blob/main/projects/uitdatabank/toc.json#L45-L231

I'm going to double check with Corneel if this is still useful though. I initially added it for Sven before we had any endpoints documented and Stoplight still used their previous design, to make it easier for him to show our schema to some partners. But it might be redundant now since the properties can now all be clearly seen on the "Event" model itself.

Reply:
Should we also order this TOC better?

Reply:
Hmm yes but I'll check with Corneel if it's actually still needed, I'll get back to you about it asap